### DISCUSSION TITLE
Skip host updates

### DISCUSSION THREAD

**Discussion by jhammond-git** [bb-Ricardo/netbox-sync#139] [Q&A] [answered]
Is there a way to skip updates to hosts in Netbox.  In other words, only create/update VMs.

**Comment by bb-Ricardo** [comment, marked as answer]
Hi,

Yes this is quite easy. Just exclude all hosts like in this example.
```ini
# This will only include/exclude the host, not the VM if Host is part of a multi host
# cluster.
host_exclude_filter = .*
#host_include_filter =
```

**Comment by jhammond-git** [comment]
Huh.  I thought I had done that but must have gotten it wrong.  Thank you!

**Reply by bb-Ricardo** [reply]
No problem. 😄
